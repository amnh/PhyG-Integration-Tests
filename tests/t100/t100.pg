--build(distance, dwag, block), inputting amino acid file
set(seed:1634561640)
read(aminoacid:"metazoa-aa-1.fas")
read(aminoacid:"metazoa-aa-2.fas")
build(distance, dWag, block, cun, displaytrees:3)
report("t100.csv", data, overwrite)
report("t100.tre", newick, overwrite)