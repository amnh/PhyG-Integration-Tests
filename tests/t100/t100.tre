(Antarctic_neomeniomorph:308.5,(Lymnaea_stagnalis:199.0,(((Littorina_littorea:108.0,(Perotrochus_lucaya:110.0,(Lottia_gigantea:97.5,(Scaphopods:148.0,(Helobdella_robusta:289.0,(Lingula_anatina:277.5,(Venerupis_philippinarum:274.0,(Greenland_neomeniomorph:144.0,Venerupis_decussatus:271.0)HTU35:40.5)HTU34:7.5)HTU33:30.5)HTU32:84.5)HTU31:80.0)HTU30:76.5)HTU29:78.5)HTU28:99.5,((Aplysia_californica:128.0,Euprymna_scolopes:235.0)HTU25:121.0,(Cerebratulus_lacteus:136.5,Octopus:36.5)HTU27:278.0)HTU36:70.0)HTU26:53.5,((Yoldia_limatula:275.0,(Nucula:241.5,(Solemya:246.5,(Mytilus_californianus:84.0,(Nautilus:252.5,(Chaetopleura_apiculata:247.5,Crassostrea_virginica:166.0)HTU45:48.0)HTU44:69.5)HTU43:60.0)HTU42:47.0)HTU41:30.0)HTU38:35.0,(Ilyanassa_obsoleta:29.0,Siphonaria_pectinata:8.0)HTU40:261.0)HTU39:33.5)HTU46:74.0)HTU37:106.0)HTU24[1578.0];
(Antarctic_neomeniomorph:335.5,((Cerebratulus_lacteus:360.5,(Greenland_neomeniomorph:214.5,Lymnaea_stagnalis:271.0)HTU45:25.5)HTU44:51.5,(((Littorina_littorea:143.5,(Perotrochus_lucaya:166.5,(Scaphopods:47.5,(Helobdella_robusta:18.5,(Venerupis_decussatus:40.5,Venerupis_philippinarum:3.5)HTU30:27.0)HTU29:303.5)HTU28:18.0)HTU27:116.0)HTU26:173.5,(Aplysia_californica:118.5,Octopus:338.0)HTU31:69.5)HTU25:81.0,(((Lingula_anatina:25.0,Yoldia_limatula:5.0)HTU36:214.5,(Euprymna_scolopes:309.0,(Lottia_gigantea:64.5,(Nucula:242.5,(Solemya:250.5,(Mytilus_californianus:85.0,(Nautilus:252.0,(Chaetopleura_apiculata:248.0,Crassostrea_virginica:166.0)HTU43:47.5)HTU42:68.0)HTU41:58.5)HTU40:49.0)HTU39:50.0)HTU38:45.5)HTU37:40.0)HTU33:43.5,(Ilyanassa_obsoleta:29.0,Siphonaria_pectinata:8.0)HTU35:215.5)HTU34:65.0)HTU46:78.0)HTU32:148.5)HTU24[1569.0];
(Antarctic_neomeniomorph:340.5,((Cerebratulus_lacteus:216.0,Lymnaea_stagnalis:176.0)HTU45:37.0,(((Littorina_littorea:102.5,(Perotrochus_lucaya:111.0,(Lottia_gigantea:97.5,(Scaphopods:149.0,(Helobdella_robusta:289.0,(Lingula_anatina:277.5,(Venerupis_philippinarum:274.0,(Greenland_neomeniomorph:144.0,Venerupis_decussatus:271.0)HTU34:40.5)HTU33:7.5)HTU32:30.0)HTU31:84.0)HTU30:80.0)HTU29:76.5)HTU28:83.5)HTU27:120.0,(Aplysia_californica:85.0,(Nucula:8.0,Octopus:100.0)HTU26:381.0)HTU35:54.0)HTU25:147.0,((Yoldia_limatula:221.0,(Euprymna_scolopes:326.0,(Solemya:250.5,(Mytilus_californianus:79.5,(Nautilus:251.5,(Chaetopleura_apiculata:248.5,Crassostrea_virginica:166.0)HTU44:48.0)HTU43:70.5)HTU42:60.0)HTU41:52.0)HTU40:68.5)HTU37:65.5,(Ilyanassa_obsoleta:29.0,Siphonaria_pectinata:8.0)HTU39:204.5)HTU38:78.5)HTU46:54.5)HTU36:152.0)HTU24[1567.0];
