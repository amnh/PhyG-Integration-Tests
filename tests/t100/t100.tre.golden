(Antarctic_neomeniomorph:327.0,((Greenland_neomeniomorph:209.0,Lymnaea_stagnalis:271.0)HTU45:20.5,(((Littorina_littorea:142.0,(Perotrochus_lucaya:167.5,(Ilyanassa_obsoleta:301.0,(Scaphopods:35.5,(Helobdella_robusta:18.5,(Venerupis_decussatus:40.5,Venerupis_philippinarum:3.5)HTU33:27.0)HTU32:303.0)HTU31:17.5)HTU30:16.0)HTU29:116.5)HTU28:165.0,(Aplysia_californica:114.0,(Nucula:7.0,(Cerebratulus_lacteus:104.5,Octopus:113.5)HTU27:59.5)HTU26:320.5)HTU34:65.0)HTU25:90.0,(Siphonaria_pectinata:201.0,((Lingula_anatina:25.0,Yoldia_limatula:5.0)HTU38:200.0,(Euprymna_scolopes:308.5,(Lottia_gigantea:67.0,(Solemya:254.5,(Mytilus_californianus:84.0,(Nautilus:251.0,(Chaetopleura_apiculata:248.5,Crassostrea_virginica:166.0)HTU44:47.5)HTU43:66.5)HTU42:60.0)HTU41:54.5)HTU40:46.5)HTU39:51.5)HTU36:59.5)HTU37:80.0)HTU46:91.5)HTU35:144.0)HTU24[1605.0];
(Antarctic_neomeniomorph:326.5,((Cerebratulus_lacteus:217.0,Lymnaea_stagnalis:176.0)HTU45:43.0,(((Littorina_littorea:114.5,(Perotrochus_lucaya:158.5,(Ilyanassa_obsoleta:289.0,(Scaphopods:139.0,(Helobdella_robusta:289.5,(Venerupis_philippinarum:274.5,(Greenland_neomeniomorph:144.0,Venerupis_decussatus:271.0)HTU35:40.5)HTU34:31.5)HTU33:92.0)HTU32:74.5)HTU31:72.0)HTU30:120.0)HTU29:119.5,((Aplysia_californica:154.5,Euprymna_scolopes:304.0)HTU26:43.0,(Nucula:39.5,(Chaetopleura_apiculata:105.0,Octopus:22.5)HTU25:50.0)HTU28:273.5)HTU36:55.5)HTU27:58.5,(Siphonaria_pectinata:242.0,((Lingula_anatina:25.0,Yoldia_limatula:5.0)HTU40:248.0,(Lottia_gigantea:67.0,(Solemya:256.0,(Mytilus_californianus:86.0,(Crassostrea_virginica:202.0,Nautilus:244.0)HTU44:67.5)HTU43:56.5)HTU42:51.0)HTU41:33.0)HTU38:40.5)HTU39:42.5)HTU46:83.5)HTU37:135.0)HTU24[1573.0];
(Antarctic_neomeniomorph:271.0,((Cerebratulus_lacteus:217.0,Lymnaea_stagnalis:176.0)HTU45:40.0,(((Littorina_littorea:118.0,(Perotrochus_lucaya:143.0,(Scaphopods:129.5,(Helobdella_robusta:282.5,(Mytilus_californianus:71.5,(Venerupis_philippinarum:274.5,(Greenland_neomeniomorph:144.0,Venerupis_decussatus:271.0)HTU34:56.0)HTU33:49.0)HTU32:59.5)HTU31:105.0)HTU30:78.5)HTU29:112.0)HTU28:99.0,((Aplysia_californica:128.5,Euprymna_scolopes:235.5)HTU25:106.5,(Nucula:54.5,Octopus:48.5)HTU27:260.0)HTU35:46.0)HTU26:74.0,(((Lingula_anatina:25.0,Yoldia_limatula:5.0)HTU40:253.5,(Lottia_gigantea:64.0,(Solemya:261.5,(Nautilus:253.0,(Chaetopleura_apiculata:247.0,Crassostrea_virginica:166.0)HTU44:47.0)HTU43:61.0)HTU42:44.5)HTU41:37.5)HTU37:35.5,(Ilyanassa_obsoleta:29.5,Siphonaria_pectinata:7.5)HTU39:244.0)HTU38:32.5)HTU46:113.0)HTU36:65.0)HTU24[1558.0];
